backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/spreadsheet/process/ExportToSpreadsheetProcess.java

```diff
@@ -92,6 +97,10 @@ else if (spreadsheetFormat == SpreadsheetFormat.CSV)
 			throw new AdempiereException("Unknown spreadsheet format: " + spreadsheetFormat);
 		}
 
+		datevAcctExport.setExportDate(SystemTime.asTimestamp());
```

Comment:
refactor this to a separate method called `updateDatevAcctExportIfNeeded` (or similar).

In that method you will:
* check if getTableName() is I_DatevAcctExport.Table_Name and if not, do nothing
* load the datevAcctExport record (as u did above), update it and save it (like you are doing it here)